new-module/liquidations.md

```diff
@@ -0,0 +1,136 @@
+---
+description: Liquidations in Angle _New_ Module
+---
+
+# 🎳 Liquidations
+
+To make sure collateral is always backing agTokens issued, Angle Vaults can get liquidated if the value of the collateral deposited gets too low compared to the value of the agTokens borrowed. This is determined by two similar parameters, the [collateral factor](/new-module/glossary.md) or the [health factor](/new-module/glossary.md). 
+
+As we will see in the [Variable liquidations amount](#variable-liquidations-amount), liquidations in Angle Borrowing module have been designed to be as much borrower friendly as possble. 
+
+**TLDR**
+* Vaults with a health factor below one risk getting liquidated
+* Liquidated vaults can lose part or all their collateral
+* Angle has special liquidations features such as a variable liquidation amount, a dynamic discount for liquidators, and a discount booster for veANGLE holders
+
+## How liquidations work
+
+### Health Factor
+Each vault has its own collateral ratio. Together with the collateral factor, they are used to compute the health factor of an open vault. For example:
+
+$$
+\texttt{HF = CR * CF} \\
+\texttt{HF} = 160/100 * 2/3 = 1.07
+$$
+
+If this health factor goes below 1, it means that the value of the collateral backing the stablecoin is too low compared to what was defined by the protocol and the position can get liquidated. 
+
+### What happens to a position getting liquidated
+
+Liquidating a position means repaying part or the entirety of the vault's debt (agTokens) against the collateral it holds (ETH or any other collateral token), usually at a discount.
+
+After getting liquidated, a vault will either be empty, or both the amount of debt and collateral will have been reduced in a way that puts the Health Factor of the vault back above a heatlhy target. 
```

Comment:
Not necessarily: health factor can be reduced and the vault may not be empty: if the liquidator does not liquidate it all

Reply:
Let's link it to the developers guide for liquidations and let's mention as well dust amounts and so on to say that a liquidation cannot put a vault in a state of liquidation